opentelekomcloud/services/ces/resource_opentelekomcloud_ces_alarmrule.go

```diff
@@ -326,37 +303,39 @@ func resourceAlarmRuleCreate(ctx context.Context, d *schema.ResourceData, meta i
 		return fmterr.Errorf(errCreationClient, err)
 	}
 
+	alarmEnabled := d.Get("alarm_enabled").(bool)
+	alarmActionEnabled := d.Get("alarm_action_enabled").(bool)
+
 	metric := getMetricOpts(d)
 	conditionListRaw := d.Get("condition").([]interface{})
 	conditionElement := conditionListRaw[0].(map[string]interface{})
```

Comment:
`conditionListRaw` length should be checked first to avoid panic

Reply:
I didn't change that part :) But I will add the check, thank you

Reply:
Yes, I just noticed that, it is not a problem now, but can become one in case of some changes, so if you're changing nearby code, it's a good idea to fix such potential problems 🙂